napytau/import_export/model/polynomial.py

```diff
@@ -0,0 +1,32 @@
+class Polynomial:
+    """
+    A class to represent a polynomial.
+    A polynomial is a mathematical expression consisting of variables and coefficients.
+    """
+
+    coefficients: list[float]
+    degree: int
```

Comment:
Why do we need both a list of coefficients and the degree? The degree should be superfluous here, since it can just be calculated with len(coefficients) - 1.

Reply:
i think while trying to understand the file formats i got this mixed up with the polynomial_count. I will remove it 👍